assets/js/googlesitekit/data/create-notifications-store.test.js

```diff
@@ -181,6 +168,7 @@ describe( 'createNotificationsStore store', () => {
 		describe( 'fetchNotifications', () => {
 			it( 'does not require any params', () => {
 				expect( () => {
+					fetchMock.get( /^\/google-site-kit\/v1\/core\/site\/data\/notifications/, { body: {}, status: 200 } );
```

Comment:
The response actually needs to be an array as `.reduce` is called on it in the reducer.